### PR TITLE
Config circleci

### PR DESCRIPTION
**Pull request by ldlsegovia**
- Run Rubocop
- Run Rspec
- Publish rubygems

image

image

### CODE REVIEW

File: lib/parxer/collections/attributes.rb [hunk 1]
```diff
@@ -6,6 +6,13 @@ def add_attribute(id, name: nil)
       end
 
       self << Parxer::Attribute.new(id, name: name)
+
+
```
Comment (monkeyci) [inline lib/parxer/collections/attributes.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: lib/parxer/collections/attributes.rb [hunk 2]
```diff
@@ -6,6 +6,13 @@ def add_attribute(id, name: nil)
...
+
```
Comment (monkeyci) [inline lib/parxer/collections/attributes.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: lib/parxer/collections/attributes.rb [hunk 3]
```diff
@@ -6,6 +6,13 @@ def add_attribute(id, name: nil)
...
+
```
Comment (monkeyci) [inline lib/parxer/collections/attributes.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: lib/parxer/collections/attributes.rb [hunk 4]
```diff
@@ -6,6 +6,13 @@ def add_attribute(id, name: nil)
...
+
```
Comment (monkeyci) [inline lib/parxer/collections/attributes.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: lib/parxer/collections/attributes.rb [hunk 5]
```diff
@@ -6,6 +6,13 @@ def add_attribute(id, name: nil)
...
+
```
Comment (monkeyci) [inline lib/parxer/collections/attributes.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra blank line detected.

File: lib/parxer/collections/attributes.rb [hunk 6]
```diff
@@ -6,6 +6,13 @@ def add_attribute(id, name: nil)
...
+
```
Comment (monkeyci) [inline lib/parxer/collections/attributes.rb]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Trailing whitespace detected.

**Review by aaguilarg — APPROVED** [review]

**Pull request merged**